From 500141665b78279e47f614dc1acfd77b845a4578 Mon Sep 17 00:00:00 2001
From: Tim Parenti <tim@timtimeonline.com>
Date: Sat, 9 Jan 2021 13:59:02 -0500
Subject: [PATCH] No leap second on 2021-06-30

Per IERS Bulletin C 61 (2021-01-07).
https://hpiers.obspm.fr/iers/bul/bulc/bulletinc.61

* leap-seconds.list: Update file from NIST, retrieved from
ftp://ftp.nist.gov/pub/time/leap-seconds.list
---
 leap-seconds.list | 8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

diff --git a/leap-seconds.list b/leap-seconds.list
index e897a86..3198d65 100644
--- a/leap-seconds.list
+++ b/leap-seconds.list
@@ -204,10 +204,10 @@
 #	current -- the update time stamp, the data and the name of the file
 #	will not change.
 #
-#	Updated through IERS Bulletin C60
-#	File expires on:  28 June 2021
+#	Updated through IERS Bulletin C61
+#	File expires on:  28 December 2021
 #
-#@	3833827200
+#@	3849638400
 #
 2272060800	10	# 1 Jan 1972
 2287785600	11	# 1 Jul 1972
@@ -252,4 +252,4 @@
 #	the hash line is also ignored in the
 #	computation.
 #
-#h 	064356a8 39268b92 76e4d5ef 3e22fae1 0cca529c
+#h 	2ab8253d d4380d28 75f01343 381504f8 8f8a4bfc
-- 
1.8.3.1

